### DISCUSSION TITLE
Verwendung der Leistungsdiagramme/Kompressorproblem

### DISCUSSION THREAD

**Discussion by fred081273** [cyberthom42/vaillant-arotherm-plus#12] [General]
Einige User haben Probleme, die in den Leistungsdiagrammen dokumentierten Werte der 75/6 zu erreichen.

Folgende Gesetzmäßigkeiten habe ich bisher verstanden
Wärmeertrag=(VL-RL)*1,16*Durchfluss
Kälteertrag=Wärmeertrag-Leistungsaufnahme
COP=Heizleistung/Leistungsaufnahme
Dann wäre die Heizleistung=Wärmeertrag

Beispiel: die 75/6 hat sollte bei 6 Grad AT eine Heizleistung von mind. 3kW erreichen.
Bei einem Durchfluss von 1200l/h ergibt sich:
Spreizung*1,16*1200=3000
Spreizung=2,15K

In meinem Fall habe ich eine Spreizung von nur 1,6K.
Was bedeutet das nun? Muss ich eine Mindestspreizung von 2,15K bei 6 Grad AT haben, damit das Gerät die Mindestleistung erreicht oder habe ich ein Problem mit meinem Kompressor, weil er die Mindestleistung nicht erreicht?
Kann es sein, dass alle, die schlechte Wärmeerträge haben, eine zu kleine Spreizung haben?
Was wäre die Lösung? Die Pumpenleistung reduzieren? Das würde aber zu weniger Volumenstrom führen und man müsste höhere Vorlauftemperaturen fahren, was zu höheren Verbrauch führt.

Also zusammengefasst: Haben wir überhaupt ein Problem mit der Wärmepumpe oder ist einfach nur die Spreizung zu gering?

**Comment by RotorMotor42** [comment]
Die Begriffe Wärmeertrag und Kälteertrag sind noch nicht ganz korrekt.
Besser ist Heizleistung und Umweltertrag.

Die Spreizung wird nicht erreicht, weil die Heizleistung zu niedrig ist.
Dies kann entweder einfach an der Messung der VL und RL Temperaturen liegen oder wirklich an eingeschränkter Leistung durch defekten Kompressor, EEV, Kondensator, Lüfter, ... usw.

**Comment by cyberthom42** [comment]
Ich habe mal in meine Diagramme geschaut. Bei mir liegt die Spreizung bei ca. 2 - 2,5K. Meine Rohdaten sind ja vorhanden. Ihr könnt euch ja mal die Spreizung anschauen.

**Reply by fred081273** [reply]
OK, danke!
Bei mir werden jetzt erstmal die Fühler geprüft.
